spec/system/admin/admin_creates_new_page_spec.rb

```diff
@@ -5,7 +5,7 @@
 
   context "when we pass through a slug param" do
     before do
-      allow(ForemInstance).to receive(:private?).and_return(true)
```

Comment:
Once we update the definition of the method this and all similar changes can be undone.